components/atom/button/src/index.scss

```diff
@@ -345,6 +349,10 @@ $icon-stroke-color: currentColor !default;
     }
   }
 
+  &--rounded {
+    border-radius: $bdrs-atom-button-rounded;
```

Comment:
shouldn't this be using the rounded border token and thats it?

Reply:
Maybe is 4 combining both usages integrated

Reply:
I don't think so cuz the width and height of the button are different (see [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius))

Reply:
right, as @andresz1 comments, `rounded` token is actually an oval if the w and h are different